### PR TITLE
Best practice edit

### PR DESCRIPTION
**Pull request by Eve832**

### CODE REVIEW

**Review by simonswine — APPROVED** [review]
LGTM

File: docs/sources/operators-guide/get-started/_index.md [hunk 1]
```diff
@@ -9,112 +9,114 @@ weight: 10
...
 
-- The written tutorial provides a series of imperative commands to start a single Phlare process.
-- The visual tutorial (in the form of a video) uses [`docker-compose`](https://github.com/grafana/phlare/tree/main/tools/docker-compose) to declaratively deploy Phlare and Grafana.
+- The **written tutorial** below provides a series of imperative commands to start a single Phlare process, or [monolith]({{< relref "../architecture/deployment-modes/index.md#monolithic-mode" >}}), which is designed for users getting started with the project. 
 
-{{< vimeo todo >}}
+- The following **video tutorial** uses [`docker-compose`](https://github.com/grafana/phlare/tree/main/tools/docker-compose) to declaratively deploy Phlare and Grafana.
 
-<br/>
+  {{< vimeo todo >}}
```
Comment (simonswine) [inline docs/sources/operators-guide/get-started/_index.md:18]: Not too sure if that video is ready somewhere?, but we shouldn't block merging this

cc @cyriltovena

Comment (simonswine) [reply docs/sources/operators-guide/get-started/_index.md]: Fixed

**Comment by 09jvilla** [comment]
Thanks Eve!

**Pull request merged**